src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.android.cs

```diff
@@ -14,11 +14,17 @@ namespace Xamarin.CommunityToolkit.UI.Views
 {
 	class SnackBar
 	{
-		internal void Show(Page sender, SnackBarOptions arguments)
+		internal void Show(VisualElement sender, SnackBarOptions arguments)
 		{
 			var view = Platform.GetRenderer(sender).View;
 			var snackBar = AndroidSnackBar.Make(view, arguments.MessageOptions.Message, (int)arguments.Duration.TotalMilliseconds);
 			var snackBarView = snackBar.View;
+
+			if (!(sender is Page))
+			{
+				snackBar.SetAnchorView(view);
+			}
```

Comment:
```suggestion
				snackBar.SetAnchorView(view);
```

Reply:
I am a bit confused are we for the brackets everywhere or no? https://github.com/xamarin/XamarinCommunityToolkit/pull/884

Reply:
I thought we are for avoiding braces if it's possible LoL

CC @pictos @jfversluis @sthewissen @jsuarezruiz

Reply:
As far I remember we don't like braces 🤔